tests/test_dse.py

```diff
@@ -1762,9 +1775,9 @@ def g3_tilde(field, phi):
 
         # Check code generation
         assert op._profiler._sections['section1'].sops == exp_ops
-        arrays = [i for i in FindSymbols().visit(op._func_table['bf0']) if i.is_Array]
+        arrays = [i for i in FindSymbols().visit(op) if i.is_Array]
```

Comment:
unnecessary variable (I know, that's my bad, not yours)

Reply:
fixed